R/gppt_graph.R

```diff
@@ -0,0 +1,266 @@
+## Detailed Level GPPT Graphs
+
+# install.packages("devtools")
+# install.packages("tidyverse")
+# devtools::install_github("NHS-South-Central-and-West/scwplot")
+# devtools::install_github("bbc/bbplot")
+
+
+library(tidyverse)
```

Comment:
Replace tidyverse with individual tidyverse packages that are used in this script. I think it should just be dplyr and ggplot2.